Source/Core/VideoBackends/D3D/D3DRender.cpp

```diff
@@ -264,19 +264,9 @@ void Renderer::UnbindTexture(const AbstractTexture* texture)
     D3D::stateman->ApplyTextures();
 }
 
-u16 Renderer::BBoxReadImpl(int index)
+std::unique_ptr<BoundingBox> CreateBoundingBox()
```

Comment:
Shouldn't this be Renderer::CreateBoundingBox?

Reply:
Heh